gensim/matutils.py

```diff
@@ -600,19 +600,13 @@ def jaccard_distance(set1, set2):
 def dirichlet_expectation(alpha):
     """
     For a vector `theta~Dir(alpha)`, compute `E[log(theta)]`.
-
+    Saves dtype of the argument.
```

Comment:
What does this comment mean? Looks out of place.

Reply:
I wanted to add some note that this function returns np.array with the same dtype as input alpha. Well, probably it's not really needed.

Reply:
If that's the intent, it's not really apparent from `Saves dtype of the argument`.